node-test/src/test/scala/org/bitcoins/node/BroadcastTransactionTest.scala

```diff
@@ -18,7 +18,7 @@ class BroadcastTransactionTest extends NodeUnitTest {
 
   /** Wallet config with data directory set to user temp directory */
   implicit override protected def config: BitcoinSAppConfig =
-    BitcoinSTestAppConfig.getNeutrinoTestConfig()
+    BitcoinSTestAppConfig.getNeutrinoWithEmbeddedDbTestConfig(pgUrl)
```

Comment:
Does this mean all our tests are using just one of the db drivers? How are we ensuring both get tested in CI?